src/pages/Base.tsx

```diff
@@ -0,0 +1,18 @@
+import { Container } from "@chakra-ui/react"
+import Navbar from "../components/UI/Navbar"
+
+// tslint:disable-next-line:no-empty-interface
+interface MyProps {}
```

Comment:
Ok I see, this is causing the error in the l'Inter. Have you tried this?https://stackoverflow.com/questions/65548388/react-propswithchildren-with-no-props-other-than-children

So the linter can be restored

Reply:
// tslint:disable-next-line:no-empty-interface
actually has no effect, so it can be removed. Since we didn't have any other interfaces I figured I could add the rule to the linter. Will try to implement the SO solution

Reply:
@gwwatkin React.PropsWithChildren<Record<never, never>> works.

No idea how or why.